app/views/drivers/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<section id="title">
+  <h1>Driver List</h1>
+  <sub>Changing The Way People Travel</sub>
+</section>
+
+<nav class="nav">
+  <li><%= link_to "HOME", root_path %></li>
+  <li><%= link_to "DRIVER", drivers_path %></li>
+  <li><%= link_to "PASSENGER", passengers_path %></li>
+  <li><%= link_to "TRIPS", trips_path %></li>
+</nav>
+
+
+<div class="buttonContainer"><%= link_to "BECOME A DRIVER", new_driver_path, method: :get, class: "button" %></div>
+
+<table style="width:100%" class="table">
+  <tr class="header_font">
+    <th>Driver ID</th>
+    <th>Name</th>
+    <th>Vin Number</th>
+    <th>Rating</th>
+    <th>Update</th>
+    <th>Delete</th>
+  </tr>
+  <% @drivers.each do |driver| %>
+  <tr>
+    <td><%= link_to driver.id, driver_path(driver.id) %></td>
```

Comment:
I suggest it makes more sense to make the driver's name the link rather than just the id.